### ISSUE TITLE
Stop using double underscore names for attributes and methods

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/coala#2724]

**Comment by RoshanAji** [comment]
I would like to have this issue assigned. Thank you

**Comment by sils** [comment]
@RoshanAji any progress?

**Comment by incorrectusername** [comment]
Is there any bear which looks for naming convensions? Like LineLengthBear checks for line length.
If there is such bear than please do tell , i can do changes to it. Or is there any other way this can be done?

**Comment by Makman2** [comment]
Yes the `FilenameBear`, but you may see that this is for filenames only ;)

This issue is more about refactoring those names, we don't want to prohibit those names. Just grep for `def __.*?__`.

**Comment by meetmangukiya** [comment]
I don't think we need the `?` in the grep regex :/ ?

**Comment by Makman2** [comment]
just to make it non-greedy @meetmangukiya ;) But right it's not really necessary :)

**Comment by incorrectusername** [comment]
We need to prevent use of `__` in future or change `__` to `_`(or only names)? 
How would you approach this? 
Till what I have understood is that we can  use  regex to prevent future use of `__`   . But there's no bear which scans the file to check for appropriate word.

**Comment by Makman2** [comment]
> We need to prevent use of __ in future or change __ to _(or only names)?

Just change.

**Comment by incorrectusername** [comment]
Okay , one last question ,in all of the files or only this one(Settings) ?😄
Can anyone assign this to me.

**Comment by Makman2** [comment]
all
